lib/metrics/version.js

```diff
@@ -8,24 +8,25 @@ const NODE_VERSION_INFO = 'nodejs_version_info';
 
 module.exports = (registry, config = {}) => {
 	const namePrefix = config.prefix ? config.prefix : '';
+	const labels = config.labels ? config.labels : {};
+	const labelNames = Object.keys(labels);
 
 	const nodeVersionGauge = new Gauge({
 		name: namePrefix + NODE_VERSION_INFO,
 		help: 'Node.js version info.',
-		labelNames: ['version', 'major', 'minor', 'patch'],
+		labelNames: ['version', 'major', 'minor', 'patch'].concat(labelNames),
```

Comment:
```suggestion
		labelNames: ['version', 'major', 'minor', 'patch', ...labelNames],
```

I believe our node version support covers this syntax? Comment applies to the other `concat` calls as well

Reply:
https://github.com/rkoval/prom-client/commit/c34d28d30997c44b8263f84b92edea9fb1c6e97d